workers/unity/Packages/com.improbable.gdk.tools/Common.cs

```diff
@@ -83,17 +83,38 @@ internal static string GetThisPackagePath()
         {
             try
             {
-                var manifest =
-                    Json.Deserialize(File.ReadAllText($"{PackagesDir}/manifest.json", Encoding.UTF8));
-                return ((Dictionary<string, object>) manifest["dependencies"]).ToDictionary(kv => kv.Key,
-                    kv => (string) kv.Value);
+                return ParseDependencies($"{PackagesDir}/manifest.json");
             }
             catch (Exception e)
             {
                 throw new ArgumentException($"Failed to parse manifest file: {e.Message}");
             }
         }
 
+        internal static Dictionary<string, string> GetPackageDependencies(string packagePath)
```

Comment:
Since this is just a wrapper around `ParseDependencies`, why not just merge the two?

Reply:
since `GetManifestDependencies` uses `ParseDependencies`, I thought to keep things separate so that if an exception throws we can tell the difference between a top-level manifest error or a package dependency error

Reply:
Hmm does that matter that we can distinguish between them in code? Surely the file path is sufficient information?

Reply:
this is true, unifying them then streamlines the `CopySchema` function too